docs/developer-docs/multi-chain/ethereum/evm-rpc/evm-rpc.mdx

```diff
@@ -93,27 +119,87 @@ JSON_RPC_SOURCE=Chain=1
+dfx canister call evm_rpc eth_sendRawTransaction "(variant {$RPC_SOURCE}, $RPC_CONFIG, \"0xf86c098504a817c800825208943535353535353535353535353535353535353535880de0b6b3a76400008025a028ef61340bd939bc2195fe537567866003e1a15d3c71ff63e1590620aa636276a067cbe9d8997f761aecb703304b3800ccf555c9f3dc64214b297fb1966a3b6d83\")" --with-cycles=$CYCLES --wallet=$WALLET
 
 # Send a raw JSON-RPC request
-dfx canister call evm_rpc request "(variant {$JSON_RPC_SOURCE}, "'"{ \"jsonrpc\": \"2.0\", \"method\": \"eth_gasPrice\", \"params\": [], \"id\": 1 }"'", 1000)" --with-cycles=$CYCLES --wallet=$WALLET
+dfx canister call evm_rpc request "(variant {$JSON_RPC_SOURCE}, "'"{ \"jsonrpc\": \"2.0\", \"method\": \"eth_gasPrice\", \"params\": [], \"id\": 1 }"'", 1000)" --with-cycles=$CYCLES --wallet=$WALLET // Local deployment
+
+dfx canister call evm_rpc request '(variant {$JSON_RPC_SOURCE}, "{\"jsonrpc\":\"2.0\",\"method\":\"eth_gasPrice\",\"params\":[],\"id\":1}",1000)' --wallet $(dfx identity get-wallet --ic) --with-cycles 1000000000 --ic // Mainnet deployment
 ```
 
+`RpcConfig` makes it possible to adjust how the RPC canister performs the HTTPS outcalls. Currently, `RpcConfig` only supports an optional `responseSizeEstimate` which can be fine-tuned to reduce the cost of the RPC requests.
+
+## Sample use case
+
+Below is a simple use case making a call to `eth_getLogs` using a piece of Rust code and the EVM RPC canister:
+
+Rust method implementation:
+
+```rust
+#[ic_cdk::update]
+async fn get_logs(get_logs_args: GetLogsArgs) -> Vec<evm_rpc::LogEntry> {
+    let rpc_providers = RpcServices::EthMainnet(Some(vec![EthMainnetService::Alchemy]));
+
+    let cycles = 10_000_000_000_000;
+    let (result,) = EVM_RPC
+        .eth_get_logs(rpc_providers, None, get_logs_args, cycles)
+        .await
+        .expect("Call failed");
+
+    match result {
+        MultiGetLogsResult::Consistent(r) => match r {
+            GetLogsResult::Ok(block) => block,
+            GetLogsResult::Err(err) => panic!("{err:?}"),
+        },
+        MultiGetLogsResult::Inconsistent(_) => {
+            panic!("RPC providers gave inconsistent results")
+        }
+    }
+}
+```
+
+Call made with `dfx`:
+
+```
+dfx canister call dappcon_backend get_logs --candid src/dappcon_backend/dappcon_backend.did  '(
+  record {
+    fromBlock = opt variant { Number = 4_365_627 : nat };
+    toBlock = opt variant { Latest };
+    addresses = vec { "0xb59f67a8bff5d8cd03f6ac17265c550ed8f33907" };
+    topics = opt vec {
+      vec { "0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef" };
+        vec { "0x00000000000000000000000000b46c2526e227482e2ebb8f4c69e4674d262e75" };
+        vec { "0x00000000000000000000000054a2d42a40f51259dedd1978f6c118a0f0eff078" };
+      };
+    };
+  },
+)'
+```
```

Comment:
I don't think this call with `dfx` adds a lot of value, and rather creates additional confusion as you need an intermediating canister such as `dappcon_backend` which hasn't been introduced. 

In general, we should focus on programmatically interacting with the EVM RPC canister from Rust, Motoko and JavaScript instead of `dfx` calls.